src/analysis/cpu_platform_profiles.md

```diff
@@ -0,0 +1,80 @@
+# CPU and platform profiles
+
+## The motivation 
+
+The computer ecosystem, especially in embedded systems is vast and growing and is thus 
+diverse and is full of trivial differences. CPUs and development boards differ by minor and sometimes 
+large differences in their design, ports, MMIO registers and other peripherals. Rizin handles these 
+differences by storing the data regarding each CPUs and platforms in [SDB](https://book.rizin.re/basic_commands/sdb.html?highlight=SDB#sdb) files in a standard format, instead of hardcoding them with each of the 
+disassembler plugins. This information will be parsed and added as flags and comments during the analysis
+loop and will show up in the disassembly and other places, making reverse engineering on those particular 
+chips ets is much easier. This also helps in easy addition of a new port, in maintenance and in user-friendliness. 
+
+## How it works
+
+### CPU profiles
+
+All the specifics pertaining to a CPU is written down in a CPU profile. It is designed in a way that allows you 
+to enter CPU specefic values like: size of the RAM (`RAM_SIZE`), size of the ROM (`ROM_SIZE`) and much more. CPU profiles 
+can be selected using the configuration variable `asm.cpu` and is loaded up during analysis (`aa`). CPU profiles also allows you to 
```

Comment:
`during the analysis (\`aa\`)` - maybe to mention somehow not only `aa` but anything beyond that as well?